tests/acceptance/gcp-secrets.bats

```diff
@@ -36,10 +36,10 @@ setup(){
     { # Braces used to redirect all setup logs.
     # 1. Write bindings file.
     cat > tests/acceptance/configs/mybindings.hcl <<EOF
-    resource "//cloudresourcemanager.googleapis.com/projects/${GOOGLE_CLOUD_PROJECT_NAME}" {
-        roles = ["roles/viewer"]
-    }
-    EOF
+resource "//cloudresourcemanager.googleapis.com/projects/${GOOGLE_CLOUD_PROJECT_NAME}" {
+    roles = ["roles/viewer"]
+}
+EOF
```

Comment:
FYI I had to shift this over like this before `bats` would find and run the tests for some reason.